ansible_collections/arista/avd/roles/eos_designs/templates/base/base.py

```diff
@@ -0,0 +1,403 @@
+
+
+class AVDStructConfig(AVDFacts):
+
+    def __init__(self, hostvars=None):
+        super().__init__(hostvars)
+
+    @cached_property
+    def _mgmt_interface_vrf(self):
+        return get(self._hostvars, 'mgmt_interface_vrf')
+
+    @cached_property
+    def _mgmt_gateway(self):
+        return get(self._hostvars, 'mgmt_gateway')
+
+    @cached_property
+    def _switch_data(self):
+        return get(self._hostvars, 'switch')
+
+    @cached_property
+    def _internal_vlan_order(self):
+        return get(self._hostvars, 'internal_vlan_order')
+
+    @cached_property
+    def _range(self):
+        return get(self._internal_vlan_order, 'range')
+
+    @cached_property
+    def _platform_settings(self):
+        return get(self._switch_data, 'platform_settings')
+
+    @cached_property
+    def router_bgp(self):
+        bgp_as = get(self._switch_data, "bgp_as")
+        if bgp_as is not None:
+            router_bgp = {'as': get(self._switch_data, 'bgp_as'),
+                          'router_id': get(self._switch_data, 'router_id'),
+                          'bgp_defaults': []}
+            for bgp_default in get(self._switch_data, 'bgp_defaults'):
+                router_bgp['bgp_defaults'].append(bgp_default)
+
+            if get(self._hostvars, 'bgp_maximum_paths') is not None and get(self._hostvars, 'bgp_ecmp') is not None:
+                router_bgp['bgp_defaults'].append(f"maximum-paths {str(get(self._hostvars, 'bgp_maximum_paths'))} "
+                                                  f"ecmp {str(get(self._hostvars, 'bgp_ecmp'))}")
+            elif get(self._hostvars, 'bgp_maximum_paths') is not None:
+                router_bgp['bgp_defaults'].append(f"maximum-paths {str(get(self._hostvars, 'bgp_maximum_paths'))}")
+            return router_bgp
+
+    @cached_property
+    def static_routes(self):
+        static_routes = []
+        mgmt_destination_networks = get(self._hostvars, 'mgmt_destination_network')
+        if mgmt_destination_networks is not None:
+            for mgmt_destination_network in self._mgmt_destination_networks:
+                static_routes.append({'vrf': self._mgmt_interface_vrf,
+                                      'destination_address_prefix': mgmt_destination_network,
+                                      'gateway': self._mgmt_gateway})
+        else:
+            static_routes.append({'vrf': self._mgmt_interface_vrf,
+                                  'destination_address_prefix': '0.0.0.0/0',
+                                  'gateway': self._mgmt_gateway})
+        return static_routes
+
+    @cached_property
+    def service_routing_protocols_model(self):
+        return 'multi-agent'
+
+    @cached_property
+    def ip_routing(self):
+        return True
+
+    @cached_property
+    def ipv6_unicast_routing(self):
+        if get(self._hostvars, 'underlay_rfc5549') is True:
+            return True
+        elif get(self._switch_data, 'underlay_ipv6') is True:
+            return True
```

Comment:
```suggestion
        return get(self._hostvars, 'underlay_rfc5549') is True or get(self._switch_data, 'underlay_ipv6') is True
```

Reply:
This change would return `False` and not `None` in the non-positive case. That might not be hitless.